propagators/propagator-aws-xray/package.json

```diff
@@ -57,6 +57,7 @@
   },
   "devDependencies": {
     "@opentelemetry/api": ">=1.0.0 <1.10.0",
+    "@opentelemetry/core": "^1.0.0",
```

Comment:
Reviewer note: This *had* been `^1.0.0` before it was moved to the core repo and then back here. The only usage is for `import { TraceState } from '@opentelemetry/core';` in tests.